{{cookiecutter.project_shortname}}/webpack.config.js

```diff
@@ -7,35 +7,49 @@ module.exports = (env, argv) => {
 
     let mode;
 
+    const overrides = module.exports || {};
 
     // if user specified mode flag take that value
     if (argv && argv.mode) {
         mode = argv.mode;
     }
 
     // else if configuration object is already set (module.exports) use that value
-    else if (module.exports && module.exports.mode) {
-        mode = module.exports = mode;
+    else if (overrides.mode) {
+        mode = overrides.mode;
     }
 
-    // else take webpack default
+    // else take webpack default (production)
     else {
-        mode = 'production'; // webpack default
+        mode = 'production';
     }
 
+    const filename = (overrides.output || {}).filename || (
```

Comment:
I find that a bit confusing with the nested ternary. Since this config will most likely be read by the user if they want learn more about how the component is built. I would put it in if else so it's more obvious.